src/install/apache-flink/install-infra.mdx

```diff
@@ -0,0 +1,6 @@
+---
+componentType: default
+headingText: Install the infrastructure agent
+---
+
+To use the Apache Flink integration, you need to also [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with specific data such as database and instance metrics.
```

Comment:
nitpicky, but the formatting of the first sentence implies you can install the integration first THEN the infra agent. I might reword to:

> Before you use the Apache flink inte, install the infra etc